src/migrations/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { Environment, Profile } from "@arkecosystem/platform-sdk-profiles";
+import { migrations } from "migrations";
+import fixtureData from "tests/fixtures/env/storage.json";
+import TestingPasswords from "tests/fixtures/env/testing-passwords.json";
+
+export const migrateProfiles = async (env: Environment, profiles: any, versionToMigrate?: string) => {
+	env.profiles().fill(profiles);
+
+	return Promise.allSettled(
```

Comment:
Using `allSettled` here can result in the migration failing and you not finding out about it unless you explicitly loop over each promise and check for the resolution status.

Reply:
@faustbrian fixed